image-loading-ktx/glide-ktx.md

```diff
@@ -0,0 +1,135 @@
+<h1 align="center">Awesome Android Extensions - <a href="https://chetangupta.net" target="_blank">AndroidBites</a>
+</h1>
+
+![awesome-android-extension-androidbites](./../androidbites_awesome_extension_banner.jpg)
+
+<p align="center">
+    A curated list of Awesome Kotlin Extensions. :octocat:
+</p>
+
+<p align="center">
+  <a href="#"><img alt="Android Language Badge" src="https://badgen.net/badge/OS/Android?icon=https://raw.githubusercontent.com/androiddevnotes/awesome-android-kotlin-apps/master/assets/android.svg&color=3ddc84"/></a>
+  <a href="#"><img alt="Kotlin Language Badge" src="https://badgen.net/badge/language/Kotlin?icon=https://raw.githubusercontent.com/androiddevnotes/awesome-android-kotlin-apps/master/assets/kotlin.svg&color=f18e33"/></a>
+</p>
+
+## :eyes: Social
+[LinkedIn](https://bit.ly/ch8n-linkdIn) | 
+[Medium](https://bit.ly/ch8n-medium) | 
+[Twitter](https://bit.ly/ch8n-twitter) | 
+[StackOverflow](https://bit.ly/ch8n-stackOflow) | 
+[CodeWars](https://bit.ly/ch8n-codewar) |
+[Portfolio](https://bit.ly/ch8n-home) |
+[Github](https://bit.ly/ch8n-git) |
+[Instagram](https://bit.ly/ch8n-insta) |
+[Youtube](https://bit.ly/ch8n-youtube) 
+
+
+## :memo: Contribution
+See [contributing.md](contribution.md)
+
+## Glide Image Loading KTX
+Useful extension for image loading using Glide
+
+### :book: Content
+* [Load Image using Glide](#ImageLoadingGlide)
+* [Load Image using Glide with placeholder](#ImageLoadingGlidePlaceholder)
+* [Load Circular Image using Glide ](#CircularImageLoadingGlide)
+* [Load Circular Image using Glide with Placeholder ](#CircularImageLoadingGlidePlaceholder)
+
```

Comment:
Hi before you go through the review have a look at this repo, it has sample with picasso image loading which is currently not tested by the API/extension design is good, I will recommend you go through them and update your extension, and if you need my help or discussion you can contact me.

https://github.com/ch8n/playground-extension-sample